.github/workflows/tests.js.yml

```
@@ -16,12 +16,12 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [ 18.x, 20.x, 21.x, 22.x]
+        node-version: [ 22.x, 24.x, 26.x ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)